docs/scripts/make_model_zoo_docs.py

```diff
@@ -98,15 +101,16 @@
         max_samples=50,
         shuffle=True,
     )
-{% elif 'segment-anything-2' in name and 'video' in name %}
+{% elif 'segment-anything' in name and 'video' in name %}
     dataset = foz.load_zoo_dataset("quickstart-video", max_samples=2)
 
     # Only retain detections in the first frame
-    for sample in dataset:
-        for frame_idx, frame in sample.frames.items():
-            if frame_idx >= 2:
-                frame.detections = None
-        sample.save()
+    (
+        dataset
```

Comment:
Idiomatic way to clear frames > 1 without reading anything into memory 🤓

Reply:
Awesome thanks!